src/Akka.TestKit.NUnit/TestKit.cs

```diff
@@ -15,6 +15,9 @@ namespace Akka.TestKit.NUnit
     /// <summary>
     /// TestKit for NUnit.
     /// </summary>
+    /// 
+    [TestFixture]
+    [FixtureLifeCycle(LifeCycle.InstancePerTestCase)]

```

Comment:
This is my fault -- I overlooked this required change in the initial PR. We need to remove this throw (here)[https://github.com/akkadotnet/Akka.TestKit.NUnit/pull/47/commits/ae5a6e00ac19e8224a17d1b99538a3cac5809160#diff-835e2f8d56950585d517422fc80535cf17ae703f07536b0850e66b3f1fc256f1R38]

Reply:
A new unit test showing that we can pass `ActorSystem` to `TestKit.ctor` would be nice as well.